extensions/authclient/BaseOAuth.php

```diff
@@ -397,7 +397,7 @@ protected function composeUrl($url, array $params = [])
     /**
      * Saves token as persistent state.
      * @param OAuthToken $token auth token
-     * @return static self reference.
+     * @return $this
```

Comment:
reach an agreement about what should be that comment with the rest of the devs and i will edit it.